### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sstzave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# 7_12_22
\ No newline at end of file
+# 7_12_22
+I want to add new line
```
Comment (sstzave) [inline README.md:2]: why?

Comment (sstzave) [reply README.md:2]: because

Comment (sstzave) [reply README.md:2]: why not?

Comment (sstzave) [reply README.md:2]: reply

Comment (sstzave) [reply README.md:2]: one more

Comment (elhowm) [reply README.md:2]: test

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
-# 7_12_22
\ No newline at end of file
+# 7_12_22
```
Comment (elhowm) [inline README.md:1]: on commit

**Pull request merged**